packages/vscode-container-client/src/test/buildContext/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM alpine

```

Comment:
We may want to generate this Dockerfile on demand for the tests; some of the static analysis tooling can be a bit boneheaded about the presence of any Dockerfiles in a repo.